tests/e2e/testcases/key/suite.go

```diff
@@ -24,9 +24,10 @@ const (
 	testKeyWith0x   = "tests/e2e/assets/test_key_0x.pk"
```

Comment:
I should move this file into something like tests/e2e/testcases/commands/avalanche_key_transfer.go
or some other convention